classes/loader.js

```diff
@@ -44,7 +46,7 @@ module.exports = class Loader {
     let coreFiles = await fs.readdirAsync(`${this.client.coreBaseDir}functions${sep}`)
       .catch(() => { fs.ensureDirAsync(`${this.client.coreBaseDir}functions${sep}`).catch(err => this.client.emit("error", this.client.funcs.newError(err))); });
     if (coreFiles) {
-      coreFiles = coreFiles.filter(file => protectedFunctions.includes(file.split(".")[0]) || !this.client.config.disabled.functions.includes(file.split(".")[0]));
+      coreFiles = coreFiles.filter(file => coreProtected.functions.includes(file.split(".")[0]) || !this.client.config.disabled.functions.includes(file.split(".")[0]));
```

Comment:
Like Ao said, you can merge this filter logic into one, opposed to two different filters.

```js
const coreFiles = coreFiles.filter(file => (coreProtected.functions.includes(file.split(".")[0]) || !this.client.config.disabled.functions.includes(file.split(".")[0])) && file.endsWith(".js"));
```